stdr_server/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ include_directories( include ${catkin_INCLUDE_DIRS})
 
 add_library(stdr_map_loader src/map_loader.cpp)
 target_link_libraries(stdr_map_loader
+    image_loader
```

Comment:
This shouldn't change. It was removed at the latest commit for a reason. Please modify the commit to exclude the change, do not commit a deletion.